skfp/fingerprints/avalon.py

```diff
@@ -12,7 +12,64 @@
 
 
 class AvalonFingerprint(FingerprintTransformer):
-    """Avalon fingerprint."""
+    """
+    Avalon fingerprint.
+
+    The implementation uses RDKit [1]_. This is a hashed fingerprint, where
```

Comment:
Generally, we don't cite RDKit, since it is quite obvious. Also, we would have to cite it basically everywhere.

Reply:
Also, maybe "uses RDKit and Avalon toolkit", since RDKit really is a wrapper for it?

Reply:
Yes it is. Painfully checked it